src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java

```diff
@@ -77,10 +87,24 @@ public SupplementaryEvidence map(
     }
 
     public List<Document> getDocsToAdd(List<Document> existingDocs, List<Document> newDocs) {
-        return newDocs
+        logDocuments("Existing docs", existingDocs);
+        logDocuments("New docs", newDocs);
+
+        final List<Document> docsToAdd = newDocs
             .stream()
             .filter(d -> existingDocs.stream().noneMatch(e -> areDuplicates(d, e)))
             .collect(toList());
+
+        logDocuments("Docs to add", docsToAdd);
+
+        return docsToAdd;
+    }
+
+    private void logDocuments(String header, List<Document> existingDocs) {
+        final String docLogs = existingDocs.stream()
```

Comment:
may be null?

Reply:
No. This method is called from `getDocsToAdd` where `existingDocs` and `newDocs` are used whithout checking for null.